com.reprezen.swagedit/resources/com.reprezen.swagedit.xml

```diff
@@ -13,5 +13,7 @@
 		<mapping pluginKey="colorTagPropertyPreferences" themeKey="foreground" />
 		<mapping pluginKey="colorFlowCharacterPreferences" themeKey="foreground" />
 		<mapping pluginKey="colorConstantPreferences" themeKey="method" />
+        <mapping pluginKey="AbstractTextEditor.Color.Foreground" themeKey="foreground"/>
```

Comment:
@tfesenko , same problem with indents here. 

More importantly, I'm inferring from this that, for some reason, the highlighting logic in effect here is not highlighting certain characters, under certain circumstances. So those characters are defaulting to the `PREFERENCE_COLOR_FOREGROUND` key defined in [`AbstractTextEditor`](https://help.eclipse.org/oxygen/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fui%2Ftexteditor%2FAbstractTextEditor.html). And that color wasn't mapped to Eclipse Color Theme, so it defaults to black. Is that right? 

The screenshot shows that it's improved, but I take it this is not the final solution.